src/test/java/io/aiven/kafka/connect/gcs/config/GcsSinkConfigTest.java

```diff
@@ -379,25 +390,89 @@ void gcsCredentialsJson() throws IOException {
         assertEquals("test-client-secret", credentials.getClientSecret());
     }
 
+    /**
+     * This test validates that the NoCredentials are used when default is specified as false. This behaviour
+     * mimics that of the Tiered Storage Manager.
+     */
     @Test
-    void gcsCredentialsExclusivity() throws IOException {
-        final Map<String, String> properties = new HashMap<>();
-        properties.put("gcs.bucket.name", "test-bucket");
+    void gcsCredentialsNoCredentialsWhenDefaultCredentialsFalse() {
+        final Map<String, String> properties = Map.of(
+            GCS_BUCKET_NAME_CONFIG, "test-bucket",
```

Comment:
as per general comment:
```suggestion
            GcsSinkConfig.GCS_BUCKET_NAME_CONFIG, "test-bucket",
```
same for others

Reply:
Done everywhere.